robot/Cumulus/resources/AdvancedMappingPageObject.py

```diff
@@ -41,21 +55,25 @@ def view_field_mappings_of_the_object(self,obj):
         self.selenium.click_link("View Field Mappings")
         self.selenium.wait_until_page_contains("Field Mappings", timeout=60)
         
-    def edit_field_mappings(self,field_name,target_field):
+    def edit_field_mappings(self,fld_name,tgt_field):
```

Comment:
I see you changed these from `field_` to `fld_`. Personally I think `field_` is more readable. Though, that's not a showstopper. Since it's your code, if you're more comfortable with abbreviations, thats fine by me.